sdcm/nemesis.py

```diff
@@ -3544,6 +3566,15 @@ def disrupt(self):
         self.disrupt_rolling_restart_cluster()
 
 
+class AuthenticatorSwitchAndRestart(Nemesis):
+    disruptive = False
```

Comment:
It's a disruptive nemesis!
It restart all the nodes in the cluster one by one.

Reply:
Will fix.